### PR TITLE
Making some responsive fixes to the people directory

### PR DESCRIPTION
**Pull request by BenCookie95**
#### Summary
Making the grid responsive

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/people/directory/people_list/people_list.tsx [hunk 1]
```diff
@@ -83,25 +83,25 @@ const PeopleList = ({
...
-                        const rowHeight = 365;
-                        const columnCount = width / columnWidth;
+                        const rowHeight = 320;
+                        const columnCount = Math.floor(width / columnWidth);
+
                         return (
                             <FixedSizeGrid
                                 ref={ref}
-                                itemData={people}
+                                itemData={{width}}
```
Comment (julmondragon) [inline components/people/directory/people_list/people_list.tsx:97]: Is this intended?

**Review by calebroseland — CHANGES_REQUESTED** [review]
Love the centered grid! Might be some further changes needed to fetching logic. (I've got a fix for the dark-themes in a pending PR)
CleanShot 2022-10-12 at 12 16 52

File: components/people/directory/directory.tsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ const Directory = () => {
             </header>
             <PeopleList
                 people={searchTerm ? searchPeople : people}
-                hasNextPage={people.length < 62}
+                hasNextPage={people.length < 102}
```
[the remark this answers is not in the document]
Comment (BenCookie95) [reply components/people/directory/directory.tsx:102]: I was waiting for an API PR which has now arrived so I will review that and we can hopefully then get rid of this hardcoded total. Will look into the search thing

File: components/people/directory/people_list/people_list.tsx (hunk 1 above)
Comment (BenCookie95) [reply components/people/directory/people_list/people_list.tsx:97]: Yep that's what it's for. Don't need the people data passed through this prop

File: components/people/directory/people_list/people_list.tsx [hunk 3]
```diff
@@ -83,25 +83,25 @@ const PeopleList = ({
...
                     isItemLoaded={isItemLoaded}
-                    itemCount={itemCount}
+                    itemCount={1000}
                     loadMoreItems={loadMoreItems}
                 >
                     {({onItemsRendered, ref}) => {
                         const columnWidth = 320;
-                        const rowHeight = 365;
-                        const columnCount = width / columnWidth;
+                        const rowHeight = 320;
```
[the remark this answers is not in the document]
Comment (BenCookie95) [reply components/people/directory/people_list/people_list.tsx:91]: Ah ok, I wasn't able to make my bio appear. Is there another server branch I need? I will adjust the height accordingly

**Comment by calebroseland** [comment]
(Also, the fetching logic as-is is perfectly fine for demo purposes; refinements there could be deferred)

File: components/people/directory/people_list/people_list.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (BenCookie95) [reply components/people/directory/people_list/people_list.tsx]: Reverted the height change

File: components/people/directory/directory.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (BenCookie95) [reply components/people/directory/directory.tsx:102]: I will fix the logic around totals in another PR, Martin has a server fix that I need to review and I will add the related frontend fixes with that

**Comment by BenCookie95** [comment]
@calebroseland & @julmondragon Can you take a look at this again please?

**Comment by calebroseland** [comment]
Will do @BenCookie95 👍

**Review by julmondragon — APPROVED** [review]
LGTM! thanks @BenCookie95

**Review by calebroseland — APPROVED** [review]

**Pull request merged**